tests/manyloc/keineschweine/enet_server/server_utils.nim

```diff
@@ -1,4 +1,6 @@
-import enet, sg_packets, estreams, md5, zlib_helpers, client_helpers, strutils,
+import ../../../../dist/checksums/src/checksums/md5
```

Comment:
Could also add `checksums` [here](https://github.com/nim-lang/Nim/blob/8e0f336f6dab061bd4dd0a07f4b436bbbe78138d/koch.nim#L559) and `import checksums/md5`